lib/pg_query/deparse.rb

```diff
@@ -332,6 +336,16 @@ def deparse_bool_expr_not(node)
       format('NOT %s', deparse_item(node['args'][0]))
     end
 
+    def deparse_boolean_test(node)
+      tests = { BOOLEAN_TEST_TRUE        => ' IS TRUE',
+                BOOLEAN_TEST_NOT_TRUE    => ' IS NOT TRUE',
+                BOOLEAN_TEST_FALSE       => ' IS FALSE',
+                BOOLEAN_TEST_NOT_FALSE   => ' IS NOT FALSE',
+                BOOLEAN_TEST_UNKNOWN     => ' IS UNKNOWN',
+                BOOLEAN_TEST_NOT_UNKNOWN => ' IS NOT UNKNOWN' }
```

Comment:
Sorry for one more nitpick - since this variable never changes, we could move it on top of the method and freeze it, to save an allocation, like this:

```ruby
BOOLEAN_TEST_TYPE_TO_STRING = {
  BOOLEAN_TEST_TRUE        => ' IS TRUE',
  BOOLEAN_TEST_NOT_TRUE    => ' IS NOT TRUE',
  BOOLEAN_TEST_FALSE       => ' IS FALSE',
  BOOLEAN_TEST_NOT_FALSE   => ' IS NOT FALSE',
  BOOLEAN_TEST_UNKNOWN     => ' IS UNKNOWN',
  BOOLEAN_TEST_NOT_UNKNOWN => ' IS NOT UNKNOWN'
}.freeze
def deparse_boolean_test(node)
  deparse_item(node['arg']) + BOOLEAN_TEST_TYPE_TO_STRING[node['booltesttype']]
end
```